packages/core/src/Slider/Slider.tsx

```diff
@@ -632,7 +632,7 @@ export const HvSlider = forwardRef<SliderRef, HvSliderProps>((props, ref) => {
             <HvLabel
               id={setId(elementId, "label")}
               className={classes.label}
-              htmlFor={setId(elementId, "slider")}
+              htmlFor={elementId}
```

Comment:
This doesn't seem right - we're be labelling the parent of the label. And `elementId` isn't being passed to an element